src/librustc_trans/trans/base.rs

```diff
@@ -2745,15 +2324,14 @@ pub fn get_item_val(ccx: &CrateContext, id: ast::NodeId) -> ValueRef {
 
             let v = match i.node {
                 ast::ItemStatic(_, _, ref expr) => {
-                    // If this static came from an external crate, then
-                    // we need to get the symbol from csearch instead of
-                    // using the current crate's name/version
-                    // information in the hash of the symbol
+                    // If this static came from an external crate, then we need to get the symbol
+                    // from csearch instead of using the current crate's name/version information
+                    // in the hash of the symbol
```

Comment:
It'll help reduce the diff if comments like this are not reformatted. I also personally dislike widening comments. Our style guidelines say that code can take up to 100 characters, not that it needs to take up to 100 characters. If no other changes are made I believe it to be best to leave it as-is.